src/components/features/vocabulary-input/index.tsx

```diff
@@ -6,9 +6,11 @@ import { Loader2, Search } from 'lucide-react'
 
 import { Input } from '@/components/ui/input'
 
-import VocabulariesServices from '@/services/csr/vocabularies'
+import VocabulariesService from '@/services/vocabulaties'
```

Comment:
![Screenshot 2024-11-24 at 21 50 25](https://github.com/user-attachments/assets/82d4f02b-7a63-4321-af6c-7a2af2a49784)
Wrong text. Please update it